Sources/HAP/HTTP/Message.swift

```diff
@@ -40,11 +42,14 @@ class Response {
             case .created: return "201 Created"
             case .accepted: return "202 Accepted"
             case .noContent: return "204 No Content"
+            case .multiStatus: return "207 Multi-Status"
             case .movedPermanently: return "301 Moved Permanently"
             case .badRequest: return "400 Bad Request"
             case .unauthorized: return "401 Unauthorized"
             case .forbidden: return "403 Forbidden"
             case .notFound: return "404 Not Found"
+            case .methodNotAllowed: return "405 Method Not Allowed"
+            case .invalidParameters: return "422 Unprocessable Entity"
```

Comment:
Should we call it `unprocessableEntity` instead of `invalidParameters`, similar to the status' description?

Reply:
Makes sense!